lib/stdlib/src/rand.erl

```diff
@@ -44,11 +47,12 @@
 %% This depends on the algorithm handler function
 -type alg_seed() :: exs64_state() | exsplus_state() | exs1024_state().
 %% This is the algorithm handler function within this module
--type alg_handler() :: #{type      := alg(),
-			 max       := integer(),
-			 next      := fun(),
-			 uniform   := fun(),
-			 uniform_n := fun()}.
+-type alg_handler() :: #{type      => alg(),
```

Comment:
@margnus1 Thx for letting me know. It was my mistake. Fix in https://github.com/erlang/otp/pull/1235/commits/6f6a3e16f366aa830b0d4acffa50969fdb5d6bce and it's committed.